src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -307,7 +307,7 @@ def github_comment_reactions():
                 extra={"organization_id": pr.organization_id},
             )
             metrics.incr("github_pr_comment.comment_reactions.missing_integration")
-            return
+            continue
```

Comment:
Are we sure about swallowing this error? What kind of situation leads to this and is there a different way we should handle it?

Reply:
we need to swallow it to continue collecting reactions from all the other comments